### DISCUSSION TITLE
Gist

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#36979] [Other Feature Feedback, Questions, & Ideas] [answered]
What are Gists? And like why do we need them?

**Comment by airtower-luna** [comment, marked as answer]
Basically they are an alternative to Pastebin etc., with the added advantage that you can clone them as Git repositories. Gists are mostly useful if you want to quickly share a few text snippets, little files, notes or similar, for which you don't want to make a full repository.

See also: https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists

**Comment by SilasRodrigues19** [comment]
@iLakshya Gist is a simple way to share code snippets and pastes with others. It is used when you need to share a sample piece of code or technique with your co-workers or friends.

See more here: [Creating gists](https://docs.github.com/en/github-ae@latest/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists)

**Comment by Khushboqof** [comment]
A gist can be a string of code, a bash script or some other small piece of data. These bits of information are hosted by GitHub as a repository. Using gist has all the benefits of utilizing a GitHub repository, and also provides a more lightweight way of utilizing GitHub's versioning features.
